ossfuzz/sndfile_alt_fuzzer.cc

```diff
@@ -0,0 +1,202 @@
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sndfile.h>
+#include <inttypes.h>
+
+typedef struct
```

Comment:
General comment: a lot of this code looks copied and pasted. Could you possibly split out the common elements from both this and the standard fuzzer and have them in common headers and common source files? Things like VIO_DATA and all the vf* functionality would be a good place to start.

Reply:
I fixed this with a `sndfile_fuzz_header.h` file. What do you think?